hazelcast/src/main/java/com/hazelcast/spi/impl/NodeEngineImpl.java

```diff
@@ -138,6 +148,18 @@ public void start() {
         proxyService.init();
         operationService.start();
         quorumService.start();
+        performanceMonitor.start();
+
+        // static loggers at beginning of file
+        performanceMonitor.register(new BuildInfoPlugin(this));
```

Comment:
Maybe we can do an automatic registration via reflection and annotations? With this solution you have to know about this part of code (and NodeEngineImpl has to know about all plugins). We could also get rid of the `ClassDataAbstractionCoupling` suppression. Could be an optimization for another PR.